src/main/java/org/molgenis/vcf/report/mapper/HtsFileMapper.java

```diff
@@ -22,11 +21,7 @@
 
   public HtsFile map(VCFHeader fileHeader, String inputFile) {
     String genomeBuild = parseGenomeBuild(fileHeader);
-    return HtsFile.newBuilder()
-        .setUri(inputFile)
-        .setGenomeAssembly(genomeBuild)
-        .setHtsFormat(HtsFormat.VCF)
-        .build();
+    return new HtsFile(inputFile, "VCF", genomeBuild);
```

Comment:
Should be ```GVCF``` for .vcf.gz (https://phenopackets-schema.readthedocs.io/en/latest/file.html#rsthtsformat)

Reply:
My comment is incorrect, GVCF does not stand for vcf.gz but for https://gatk.broadinstitute.org/hc/en-us/articles/360035531812-GVCF-Genomic-Variant-Call-Format